phpunit/block-supports/layout-test.php

```diff
@@ -61,7 +61,7 @@ public function test_outer_container_not_restored_for_non_aligned_image_block_wi
 		$block_content = '<figure class="wp-block-image size-full"><img src="/my-image.jpg"/></figure>';
 		$expected      = '<figure class="wp-block-image size-full"><img src="/my-image.jpg"/></figure>';
 
-		$this->assertSame( $expected, gutenberg_restore_image_outer_container( $block_content, $block ) );
+		$this->assertEqualHTML( $expected, gutenberg_restore_image_outer_container( $block_content, $block ) );
```

Comment:
From the test failures it looks like `assertEqualHTML` isn't defined. I think it only exists in the wp-develop tests so far.

Reply:
oh darn! that’s too bad. yeah this was from `wordpress/wordpress-develop` first and I didn’t realize these haven’t migrated over. that could be helpful to have here. perhaps I will look into porting it, or @sirreal or @ockham…